Update watchOS and visionOS to officially supported (#36219)
DataDog/documentation: content/en/real_user_monitoring/application_monitoring/ios/_index.md

Reviewer: ```suggestion
## Start monitoring Apple applications (iOS, iPadOS, tvOS, watchOS, and visionOS)
```

Reviewer: Maybe include macOS too?

Reviewer 2: I wouldn't add macOS as it is not officially supported.

Reviewer 3: ```suggestion
## Start monitoring Apple platform applications
```

Reviewer: ```suggestion
To get started with RUM for the Apple ecosystem, create an application and configure the iOS SDK. The iOS SDK covers all platform listed in [Supported Versions]({add link to the "Supported Versions" page})
```

Reviewer 3: To get started with RUM for Apple platforms, create an application and configure the iOS SDK. the Datadog iOS SDK covers all platforms listed in [Supported Versions][1].

Reviewer 3: ```suggestion
{{< /whatsnext >}}

[1]: /real_user_monitoring/application_monitoring/ios/supported_versions
```

Reviewer 3: should this be updated to include the other platforms?
```suggestion
  <u>Integrated Libraries</u>: Import integrated libraries you can use for your iOS and tvOS applications.{{< /nextlink >}}
```

---

> title: iOS, tvOS, watchOS, and visionOS Monitoring

Reviewer 3: ```suggestion
title: Apple Platform Monitoring
```

> description: Collect RUM and Error Tracking data from your iOS projects.

Reviewer 3: ```suggestion
description: Collect RUM and Error Tracking data from your Apple platform applications.
```

  {{< nextlink href="/real_user_monitoring/application_monitoring/ios/mobile_vitals">}}<u>Mobile Vitals</u>: View mobile vitals, which help compute insights about your mobile application.{{< /nextlink >}}
  {{< nextlink href="/real_user_monitoring/application_monitoring/ios/web_view_tracking/?tab=ios">}}<u>Web View Tracking</u>: Monitor web views and eliminate blind spots in your mobile applications.{{< /nextlink >}}
  {{< nextlink href="/real_user_monitoring/application_monitoring/ios/integrated_libraries">}}

>   <u>Integrated Libraries</u>: Import integrated libraries you can use for your iOS and tvOS applications.{{< /nextlink >}}

Reviewer 3: ```suggestion
  <u>Integrated Libraries</u>: Import integrated libraries you can use for your Apple platform applications.{{< /nextlink >}}
```
